torch/utils/data/dataloader.py

```diff
@@ -238,10 +244,12 @@ class DataLoader(object):
         pin_memory (bool, optional)
     """
 
-    def __init__(self, dataset, batch_size=1, shuffle=False, sampler=None,
-                 num_workers=0, collate_fn=default_collate, pin_memory=False):
+    def __init__(self, dataset, batch_size=1, drop=False, shuffle=False,
```

Comment:
Move that argument to the end of the arg list. Otherwise it breaks backward compatibility.
Also, I'd rename `drop` to `drop_last`.